src/ansys/dpf/core/operators/mapping/fft.py

```diff
@@ -269,26 +289,47 @@ def inplace(self):
         return self._inplace
 
     @property
-    def calc_conjugated_coef(self):
-        """Allows to connect calc_conjugated_coef input to the operator.
+    def force_fft_points(self):
+        """Allows to connect force_fft_points input to the operator.
+
+        Explicitely define number of fft points to
+        either rescope or perform zero
+        padding.
+
+        Parameters
+        ----------
+        my_force_fft_points : int
+
+        Examples
+        --------
+        >>> from ansys.dpf import core as dpf
+        >>> op = dpf.operators.mapping.fft()
+        >>> op.inputs.force_fft_points.connect(my_force_fft_points)
+        >>> # or
+        >>> op.inputs.force_fft_points(my_force_fft_points)
+        """
+        return self._force_fft_points
+
+    @property
+    def cutoff_frequency(self):
+        """Allows to connect cutoff_frequency input to the operator.
 
-        True to trigger calculation of conjugated
-        coefficient (real to complex fft
-        only).
+        Restrict output frequency output up to this
+        cutoff frequency
```

Comment:
```suggestion
        This property restricts output frequency output up to this
        cutoff frequency
```
Not sure why there are two "outputs" in "... output frequency output...."